examples/cpp/onedal/standalone/pca.cpp

```diff
@@ -32,9 +32,9 @@ int main(int argc, char const *argv[]) {
         -4.f, 3.f,  0.f
     };
 
-    const auto table = dal::create_table(data, row_count, column_count);
+    const auto table = dal::make_table(data, row_count, column_count);
 
-    const auto params = pca::params()
+    const auto params = pca::params<>()
```

Comment:
What the problem with `<>`? C++17 allows class template argument deduction. I think, problem with the old GCC, try GCC 8 or 9.